src/Security/Security.php

```diff
@@ -1009,10 +1105,11 @@ public static function check_default_admin($username, $password)
 
     /**
      * Check that the default admin account has been set.
+     * @todo Check if we _actually_ only want this to work on dev
```

Comment:
No, ops need this for live / testing instances too. Please remove the dev-only exemption.